add entrypoint execute fuzzer (#182)
lambdaclass/starknet_in_rust: fuzzer/README.md

# To run this fuzzer

## 1. Create a container to run it 

> use user guide section on this [repository](https://github.com/lambdaclass/fuzzing_examples) to create a container with the dockerfile in it

Reviewer: ```suggestion
Please refer to the user guide section in this [repository](https://github.com/lambdaclass/fuzzing_examples) to create a container with the included Dockerfile.
```

Author: done!

## 2. Run the fuzzer 

1. clone the proyect in the container
2. start the enviroment following the proyect main README
3. run the command `HFUZZ_RUN_ARGS="-n 1" cargo hfuzz run fuzzer` within the fuzzer folder

> 4. the crashes found will be stored in the hfuzz_workspace folder along with the crash report and all the inputs used

Reviewer: Please capitalize this.

Author: done!

> As this fuzzer compiles several files while running it causes a crash, its already documented so unless the fuzzer marks the crash fund is unique: 1 , the error triggered doesn´t reproduce a real error.

Reviewer: Maybe we could use a more formal approach like:
```suggestion
Since this fuzzer compiles multiple files while it runs, crashes may occur. However, this issue has already been documented. Unless the fuzzer flags the crash as unique (i.e., marked as "1"), the error triggered may not indicate a genuine problem.
```

Author: the previous changes in the fuzzer avoid this behavior so this comment its no longer needed
